presto-docs/src/main/sphinx/release/release-0.204.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.203
+=============
+
+General
+-------
+* Fix a memory tracking bug that causes the memory to be overestimated for scan operators when reading ORC files.
```

Comment:
The engine doesn't know anything about ORC files. Phrase it like this:
```
* Fix a memory tracking bug that causes the memory to be overestimated for scan operators.
  In particular, this affects reading ORC files in the Hive connector.

Reply:
I think this is an issue specifically for ORC, not for scan operators. So, if anything, it's the opposite -- the bug in ORC affects the memory estimation for the scan operator.

Reply:
Yes, this is specifically for ORC files, not scans in general.